app/views/coronavirus_form/nhs_number.html.erb

```diff
@@ -15,20 +15,35 @@
       "novalidate": "true"
     ) do %>
 
+    <%= render "govuk_publishing_components/components/title", {
+      title: t("coronavirus_form.questions.nhs_number.title"),
+      margin_top: 0,
+      margin_bottom: 3,
+    } %>
+
+    <%= render "govuk_publishing_components/components/hint", {
```

Comment:
hint component is not meant to be used in solitude, only part of a fieldset/input field to help describing the input itself. I would recommend outputting it as plain text `<%= t("coronavirus_form.questions.nhs_number.hint") %>` and wrap [each content line in the yml](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/328/files#diff-37c9b835927e3c363588f172de9bb6faR362-R365) in a `<p class="govuk-body">`. aside from this suggestion, frontend looks good 👍

Reply:
Thanks @alex-ju - have updated as requested.